compiler+runtime/src/cpp/main.cpp

```diff
@@ -231,6 +231,9 @@ try
   using namespace jank;
   using namespace jank::runtime;
 
+  /* Set locale */
```

Comment:
This is what I call a "what" comment, which is the type of comment to most readily avoid. It says what the code does. In this case, and in most cases with "what" comments, the code is obvious. Instead, comments should generally answer other questions, such as "why", "how", and "when".

For an example, take a look at the comment right below this code, regarding the GC. We don't say "Enable GC interior pointers". That's what the code does. Instead, we provide context about why the code is there and when other GC-related code might happen.

In this case, the useful comment is to explain "why" we're setting this locale. What happens if we don't?